rails/app/views/browse/_show.html.haml

```diff
@@ -51,9 +51,9 @@
     %div{:style=>'float:left;padding:3px'}
       = render :partial => '/browse/preview', :locals => {:material => parent_material.material, :current_visitor => current_visitor}
     -if external
-      - if material.launch_url  && ((current_visitor.has_role? 'admin','manager') || (material.author_email == current_visitor.email && (current_visitor.has_role? 'author')))
+      - if material.author_url  && ((current_visitor.has_role? 'admin','manager') || (material.author_email == current_visitor.email && (current_visitor.has_role? 'author')))
         %div{:style=>'float:left;padding:3px'}
-          = link_to "Edit", matedit_external_activity_url(material,:iFrame => false), :class=>"button", :target => "_blank"
+          = link_to "Edit", material.author_url + '?iFrame=false', :class=>"button", :target => "_blank"
```

Comment:
This seems wrong. The `matedit_external_activity_url` being replaced is an internal page that would then redirect to LARA if iframe was false. Or it would render the matedit page and iframe in LARA.

Did you try leaving this as it was and see if the matedi page would do the right thing with the author_url?
Since the matedit action of the external_activities_controller is updated in this PR, I'd think this code would work without changes. 

If we just take them directly to material.author_url, that wouldn't add in the 'domain' and 'domain_uid' params that are needed so LARA automatically logs in the user so they can start editing without seeing a permission denied message if they were not logged in to LARA. You can see those params being added in the matedit action.